src/EnergyPlus/MixedAir.cc

```diff
@@ -568,6 +568,34 @@ void SimOAComponent(EnergyPlusData &state,
             SimDXCoolingSystem(state, CompName, FirstHVACIteration, AirLoopNum, CompIndex);
         }
         OACoolingCoil = true;
+    } else if (CompTypeNum == ComponentType::CoilSystemWater) { // "CoilSystem:Cooling:Water"
+        if (state.dataAirLoop->OutsideAirSys(OASysNum).compPointer[CompIndex] == nullptr) {
+            UnitarySystems::UnitarySys thisSys;
+            state.dataAirLoop->OutsideAirSys(OASysNum).compPointer[CompIndex] =
+                thisSys.factory(state, DataHVACGlobals::UnitarySys_AnyCoilType, CompName, false, 0);
+            UnitarySystems::UnitarySys::checkUnitarySysCoilInOASysExists(state, CompName, 0);
+        }
+        if (Sim) {
+            bool HeatingActive = false;
+            bool CoolingActive = false;
+            Real64 OAUCoilOutTemp = 0.0;
+            bool ZoneEquipFlag = false;
+            Real64 sensOut = 0.0;
+            Real64 latOut = 0.0;
+            state.dataAirLoop->OutsideAirSys(OASysNum).compPointer[CompIndex]->simulate(state,
+                                                                                        CompName,
+                                                                                        FirstHVACIteration,
+                                                                                        AirLoopNum,
+                                                                                        CompIndex,
+                                                                                        HeatingActive,
+                                                                                        CoolingActive,
+                                                                                        CompIndex,
+                                                                                        OAUCoilOutTemp,
+                                                                                        ZoneEquipFlag,
+                                                                                        sensOut,
+                                                                                        latOut);
+        }
+        OACoolingCoil = true;
```

Comment:
Adds a new calling point for "CoilSystem:Cooling:Water" placed in OA system. If the pointer is null, then invokes the new instantiates this coil system as UnitarySystem.